pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -16,6 +16,8 @@
 , clblast
 
 , blasSupport ? builtins.all (x: !x) [ cudaSupport metalSupport openclSupport rocmSupport vulkanSupport ]
+, openblas
```

Comment:
Or just `blas`?

Reply:
I added a second commit to switch from openblas to blas.